MC/bin/o2dpg_sim_workflow.py

```diff
@@ -442,7 +442,7 @@ def createRestDigiTask(name, det='ALLSMALLER'):
    workflow['stages'].append(ITSTPCMATCHtask)
 
    TRDTRACKINGtask = createTask(name='trdreco_'+str(tf), needs=[TRDDigitask['name'], ITSTPCMATCHtask['name'], TPCRECOtask['name'], ITSRECOtask['name']], tf=tf, cwd=timeframeworkdir, lab=["RECO"])
-   TRDTRACKINGtask['cmd'] = 'echo "would do TRD tracking"' # 'o2-trd-global-tracking'
+   TRDTRACKINGtask['cmd'] = 'o2-trd-global-tracking ' + getDPL_global_options(bigshm=True, nosmallrate=False) + ' --disable-mc' # TRD tracker cannot handle MC labels yet
    workflow['stages'].append(TRDTRACKINGtask)
```

Comment:
Thanks. I'll test it tomorrow. Is there also some consumer of these tracks (global track matching / AOD)?